icu4c/APIChangeReport.md

```diff
@@ -0,0 +1,870 @@
+
+
+<!--
+ Copyright (C) 2016 and later: Unicode, Inc. and others.
```

Comment:
should be 2019 and the real copyright symbol

Reply:
fixed in #807 , will regen